.eslintrc

```diff
@@ -3,72 +3,5 @@
     "es6": true,
     "node": true
   },
-  "parser": "@typescript-eslint/parser",
-  "parserOptions": {
-    "ecmaVersion": 2021,
-    "project": true
-  },
-  "plugins": ["import", "@typescript-eslint"],
-  "extends": [
-    "eslint:all",
-    "plugin:import/recommended",
-    "plugin:import/typescript",
-    "plugin:@typescript-eslint/strict-type-checked",
-    "plugin:@typescript-eslint/stylistic-type-checked",
-    "prettier"
-  ],
-  "rules": {
-    // This is personal preference. Ternary is helpful for shorten conditional code.
-    // https://eslint.org/docs/latest/rules/no-ternary
-    "no-ternary": "off",
-    // There are many cases that alphabetic sort not the best solution.
-    // https://eslint.org/docs/latest/rules/sort-vars
-    "sort-vars": "off",
-    // There are many cases that alphabetic sort not the best solution.
-    // https://eslint.org/docs/latest/rules/sort-keys
-    "sort-keys": "off",
-    // This is a discord bot project, which console will not be viewable by user
-    // https://eslint.org/docs/latest/rules/no-console
-    "no-console": "off",
-
-    // In ES6, make variable declaration in one line doesn't make sense. At
-    // opposite, we should avoid it.
-    // https://eslint.org/docs/latest/rules/one-var
-    "one-var": [
-      "error",
-      {
-        "let": "never",
-        "const": "never"
-      }
-    ],
-
-    // https://eslint.org/docs/latest/rules/no-void
-    "no-void": [
-      "error",
-      {
-        // We need this to do asynchronous in event handler.
-        "allowAsStatement": true
-      }
-    ],
-
-    // https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/extensions.md
-    "import/extensions": [
-      "error",
-      {
-        "js": "never",
-        "ts": "never"
-      }
-    ]
-  },
-  "settings": {
-    "import/resolver": {
-      "node": {
-        "extensions": [".js", ".ts", ".d.ts"]
-      },
-      "typescript": {
-        "project": "./tsconfig.json",
-        "alwaysTryTypes": true
-      }
-    }
-  }
+  "extends": ["@oysterlee/eslint-config-ts"]
```

Comment:
I am intented to use a very strict eslint rule for this project, you could disable/configure some of the rules if it is reasonable.

Reply:
But I will be able to help you do the clean up and avoid the eslint warning if you are busy to do that